apps/condo/lang/en.json

```diff
@@ -371,7 +371,7 @@
   "pages.users.PageTitle": "Users",
   "pages.users.status.Accepted": "Invite is accepted",
   "pages.users.status.NotActive": "Not active",
-  "pages.users.status.Rejected": "Invite is rejected",
+  "pages.users.status.Rejected": "The user rejected the invitation, you cannot invite him again",
```

Comment:
May be "This user" instead of double the...the?
But, may be it's ok 🤔

Reply:
User has rejected the invitation